code/modules/clothing/suits/cloaks.dm

```diff
@@ -14,39 +14,39 @@
 	user.visible_message("<span class='suicide'>[user] is strangling themself with [src]! It looks like they're trying to commit suicide.</span>")
 	return(OXYLOSS)
 
-/obj/item/clothing/cloak/hos
+/obj/item/clothing/suit/cloak/hos
 	name = "head of security's cloak"
 	desc = "Worn by Securistan, ruling the station with an iron fist. It's slightly armored."
 	icon_state = "hoscloak"
-	armor = list(melee = 10, bullet = 10, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+	armor = list(melee = 20, bullet = 15, laser = 15, energy = 10, bomb = 25, bio = 0, rad = 0)
 
-/obj/item/clothing/cloak/qm
+/obj/item/clothing/suit/cloak/qm
 	name = "quartermaster's cloak"
 	desc = "Worn by Cargonia, supplying the station with the necessary tools for survival."
-
+	armor = list(melee = 0, bullet = 15, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
```

Comment:
This change doesn't even make any sense. Why would the QM's cloak have bullet resist on it? He's nowhere near bullets 99% of the time. Melee? At least that somewhat makes sense, due to mining, but...really...why does the QM need armor, at all?